android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/Retailer.kt

```diff
@@ -45,27 +52,107 @@ class Retailer {
     }
 
     @OptIn(ExperimentalCoroutinesApi::class)
-    fun account(
-        client: AccountLinkingClient,
-        retailerId: Int,
-        email: String,
-        password: String,
-    ): CompletableDeferred<Boolean> {
+    fun account(call: PluginCall): CompletableDeferred<Boolean> {
```

Comment:
The PluginCall already work as callback. The method doesn't need to return anything. Check Email login method.